src/antsibull_core/ansible_core.py

```diff
@@ -252,43 +252,31 @@ async def checkout_from_git(download_dir: str, repo_url: str = _ANSIBLE_CORE_URL
     :kwarg: repo_url: The url to the git repo.
     :return: The directory that ansible-core has been checked out to.
     """
-    loop = asyncio.get_running_loop()
     ansible_core_dir = os.path.join(download_dir, 'ansible-core')
-    await loop.run_in_executor(None, sh.git, 'clone', repo_url, ansible_core_dir)
+    await async_log_run(['git', 'clone', repo_url, ansible_core_dir])
 
     return ansible_core_dir
 
 
-@lru_cache(None)
+@cache
 async def create_sdist(source_dir: str, dest_dir: str) -> str:
```

Comment:
This function isn't used anywhere (antsibull has its own code to build sdist and wheels). IMO, we should remove it in antsibull-core v2. What do you think?

Reply:
I'm fine with removing it (as long as that's documented, and as long as the integration tests with the older antsibull versions do not fail - I don't think they should, but who knows :shrug:).

Reply:
Doh, this is used later in this same file...